core/src/main/java/org/mskcc/cbio/portal/util/SessionServiceUtil.java

```diff
@@ -38,7 +38,15 @@
 import org.apache.commons.io.IOUtils;
 import org.apache.commons.httpclient.HttpException;
 
+
```

Comment:
we can get rid of this file once we have SPA. Code here is kinda hacky with some duplication, but we'll remove the file later so not a big deal imo